tests/javaobject/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ include(UseJava)
 
 file(GLOB_RECURSE SOURCES "*.java")
 
+# Check for versions of GEODE that don't support AuthenticationExpiredException
+if (${Geode_VERSION} VERSION_LESS ${GEODE_AUTHEXPIREDEXCEPTION_VERSION})
```

Comment:
Use version literal, `"1.15.0"`, here.

Reply:
It needs to be a variable that we can override on the cmake config command line, for closed source builds:
   cmake ... -DGEODE_AUTHEXPIREDEXCEPTION_VERSION=9.15.0

Reply:
No longer need to define the variable. Switched to searching for the Java class.